addons/account/models/company.py

```diff
@@ -231,6 +231,28 @@ class ResCompany(models.Model):
     # Autopost Wizard
     autopost_bills = fields.Boolean(string='Auto-validate bills', default=True)
 
+    def _get_default_account_price_include(self):
+        return 'tax_excluded'
```

Comment:
Why? I don't find anything related to POS in the specs.

Reply:
Here, point 4.

![immagine](https://github.com/user-attachments/assets/f194f9da-3766-47be-9520-a809989a7859)